lib/audited/auditor.rb

```diff
@@ -285,11 +285,10 @@ def auditing_enabled
 
       def run_conditional_check(condition, matching: true)
         return true if condition.blank?
-
         return condition.call(self) == matching if condition.respond_to?(:call)
-        return send(condition) == matching if respond_to?(condition.to_sym)
+        return send(condition) == matching if respond_to?(condition, include_all: true)
 
-        true
+        false
```

Comment:
why did you change the default to false?

Reply:
With the second parameter as `true` on `respond_to?` at last return, the default will never be raised I think. Anyway, I'll rollback it.